src/licenses/index.ts

```diff
@@ -1,221 +1,147 @@
-import { Base } from '../base';
-import { RESOURCE_NAMES } from '../constants';
-import {
-  ICheckLicensesCapabilities,
-  ILicense,
-  ICreateAdhocLicenseInput,
-  IUpdateLicenseInput,
-  IUpdateManyLicenseInput,
-  ILicenseCountResponse,
-  Status,
-  LicenseCancelManyBody,
-  LicenseGetByPurchaserOptions,
-  LicenseGetUsage,
-} from '../types';
-import jsrsasign from 'jsrsasign';
+import { ApiRequest, TVersion, Version } from "..";
+import { CheckLicenseInput, CheckLicensesCapabilitiesResponse, CreateAdhocLicenseInput, GetAllLicensesResponse, GetLicenseCountInput, GetLicenseOptions, GetPurchasersLicensesInput, License, GetLicenseCountResponse, UpdateManyLicenseInput } from "@/src/types";
+import { v2LicenseMethods } from "./v2";
 
-/**
- * Salable Node SDK License Class
- *
- * Contains the Salable license methods
- */
-export default class Licenses extends Base {
+export type LicenseVersions = {
+  [Version.V2]: {
+    /**
+    *  Get all licenses
+    * 
+    * @param {GetLicenseOptions} options
+    * @param {GetLicenseOptions} options.status - The status of the licenses, can be one of: "ACTIVE" "CANCELED" "EVALUATION" "SCHEDULED" "TRIALING" "INACTIVE"
```

Comment:
There is no longer an `"EVALUATION"` status. Will need to double-check the rest.